app/pages/gsoc/gsoc2022/[id].js

```diff
@@ -1,11 +1,15 @@
 import Head from "next/head";
+import Image from "next/image";
 import { fetchAPI } from "../../../lib/api";
 import { contributorList } from "../../../lib/leaderboard";
-import { Container, Col } from "react-bootstrap";
+import { Container, Col, Table } from "react-bootstrap";
 import styles from "../../../styles/Leaderboard.module.css";
```

Comment:
This is part of the component, it should not need to be imported here.

Reply:
I have hidden these inside the leaderboard component now. No CSS leaks on the page anymore :)